index.html

```diff
@@ -24,10 +24,37 @@ <h2><input type="text" class="nes-input" placeholder="what needs to be done" v-o
     </li>
   </ul>
 
-  <center><button type="button">Active</button>
-  <button type="button">Completed</button></center>
+  <center>
+  <button v-on:click="showCompleted">Completed</button>
+  <button v-on:click="showActive">Active</button>
 </div>
 
+<ul>

```

Comment:
make the <ul> only show up if this.showActive is true:
`<ul v-if="showActive">`